owncloudApp/src/main/res/values/colors.xml

```diff
@@ -67,4 +67,8 @@
     <!-- Bottom Sheet Fragment -->
     <color name="bottom_sheet_fragment_item_color">#6E758C</color>
     <color name="bottom_sheet_fragment_title_color">#6E758C</color>
-</resources>
\ No newline at end of file
+
+    <!-- Spaces -->
+    <color name="spaces_card_background_color">#edf3fa</color>
+    <color name="spaces_card_default_image_color">#4c5f79</color>
```

Comment:
We may need to move them to setup xml because that blue is from oC

Reply:
Ok, moved!